volume/mounts/windows_parser.go

```diff
@@ -398,6 +398,9 @@ func (p *windowsParser) parseMountSpec(cfg mount.Mount, destRegex string, conver
 			if cfg.VolumeOptions.NoCopy {
 				mp.CopyData = false
 			}
+			if cfg.VolumeOptions.SubPath != "" {
+				mp.SubPath = strings.Replace(cfg.VolumeOptions.SubPath, `/`, `\`, -1)
```

Comment:
I see there's other lines like this in this file, so probably this is correct (just thinking about the LCOW situation, but not sure if we hit this part of the code for that)

Reply:
It seems the LCOW parser uses the windows parser, except for a different regular expression for the destination:

https://github.com/moby/moby/blob/6d18c6a0629581ba35286fc04d6a8462ad20883a/volume/mounts/lcow_parser.go#L20-L22

https://github.com/moby/moby/blob/6d18c6a0629581ba35286fc04d6a8462ad20883a/volume/mounts/lcow_parser.go#L32-L33


I will update the test case in `parser_test.go` to cover the conversion of the path separator.